contracts/cheddar_5x5_tic_tac_toe/src/token_receiver.rs

```diff
@@ -66,41 +57,18 @@ impl FungibleTokenReceiver for Contract {
 }
 
 impl Contract {
-    pub(crate) fn internal_make_available(
+    pub(crate) fn make_deposit(
```

Comment:
```suggestion
    pub(crate) fn try_deposit_cheddar(
```

Reply:
also let's add a doc comment to this function

Reply:
DONE